projects/pt1/python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -6453,3 +6453,66 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: AtenNonzero1DDynamicModule())
 def AtenNonzero1DDynamicModule_basic(module, tu: TestUtils):
     module.forward(torch.tensor([0, 0, 1, 1, 0, 0], dtype=torch.bool))
+
+
+# ==============================================================================
+
+
+class AtenSymConstrainRange(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([None, ([-1], torch.int, True)])
+    def forward(self, x):
+        a = x.item()
+        torch._check_is_size(a)
+        torch.ops.aten.sym_constrain_range(a, max=5)
+        return a
+
+
+@register_test_case(module_factory=lambda: AtenSymConstrainRange())
+def AtenSymConstrainRange_basic(module, tu: TestUtils):
+    module.forward(torch.tensor(4))
+
+
+# ==============================================================================
+
+
+class AtenSymConstrainRangeForSize(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([None, ([-1], torch.int, True)])
+    def forward(self, x):
+        a = x.item()
+        torch._check_is_size(a)
+        # max should be > 2
```

Comment:
@praveen-g-ctt, I don't think this comment is needed here. It should only be part of the lowering.